src/features/collection/state/collectionState.ts

```diff
@@ -84,6 +94,14 @@ export const collectionReducer = createReducer(initialState, {
   'API_RUNNING_FAILURE': (state, action) => {
     state.runningLoading = false
   },
+  'API_VERSIONINFO_REQUEST' : (state, action) => {
+    state.pendingIDs.push(action[CALL_API].body.initID)
+  },
+  'API_VERSIONINFO_SUCCESS':(state, action) => {
+    const versionInfo: VersionInfo = action.payload.data;
+    state.pendingIDs = state.pendingIDs.filter(id => id !== versionInfo.initID)
+    state.collection[versionInfo.initID] = newVersionInfo(versionInfo)
+  },
```

Comment:
should add a failure case here to remove the initID from the `pending` list. If this causes an infinite loop we should talk over how to fix it.

Reply:
@ramfox , it causes infinite loop.

Reply:
okay fab. After looking into this more, I think we have some bigger structural issues to deal with that will help fix this problem, so we should just leave it alone for now and move on.